src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -274,3 +274,8 @@
 (defn get-balance-for-chain
   [data chain-id]
   (some #(when (= chain-id (:chain-id %)) %) (vals data)))
+
+(defn string-to-vector
```

Comment:
this utils file is starting to get oversized and seemingly is starting to have a big mixed bag of functions. We should really clean it up and have like
utils/address
utils/balances
or some cleaner separation of utilities before this grows any further.

Following on that point, `string-to-verctor` has very little to do with the wallet and should take place in a better suited file. Probably in some status-im/common or in src/utils somewhere. 
@ulisesmac perhaps you know of somewhere in the codebase better suited for this helper?

Reply:
Moved it to `utils/strings.cljs`

Reply:
Please use safe split too, the clojure string methods are causing a lot of crashes all over the app 🥲

Reply:
Just a side opinion, but I think there's kind of a misunderstanding about Clojure string functions overall in the repo. We shouldn't even need safe wrappers because we should know string functions should check for nils first, have default values or use something like `some->`.

Reply:
sure, but why wouldn't we just abstract that away?

Reply:
@ilmotta 

Well, as Clojurists you get used to use `some->` on string functions you know can break, but the safety ckecks in `clojure.string` are inconsistent.

Sometimes it's checked:

```clojure
(defn ^boolean blank?
  "True is s is nil, empty, or contains only whitespace."
  [s]
  (gstring/isEmptyOrWhitespace (gstring/makeSafe s))) ;<- makeSafe
```

That function checks and casts if needed, that's the same we would do:

```js
goog.string.makeSafe = function(obj) {
  'use strict';
  return obj == null ? '' : String(obj);
};

```

but then `lower-case` checks nothing:
```clojure
(defn ^string lower-case
  "Converts string to all lower-case."
  [s]
  (.toLowerCase s))
```

So I'd say the core is inconsistent, although I don't like to wrap all string functions, I think we should address in some way this clojure.string behavior

Reply:
Thanks @ulisesmac, it is indeed not needed